Preference Access SSN Matching/LETTERSPECS/PREFACCESS.SSNMATCHING.CFG

```diff
@@ -0,0 +1,155 @@
+*******************************************************************************
+* PREFACCESS.SSNMATCHING.CFG
+*
+* This letterfile contains parameters for the RB.PREFACCESS.SSNMATCHING
+* specfile. Each line starting with an asterisk will be considered a comment
+* so feel free to add additional lines as necessary.  Each  line in this
+* letterfile cannot be longer than 126 characters. Each parameter setting
+* consists of a parameter name followed by a colon and the parameter value
+*  (for example: 'CT: 1')
+* Where instructions call for entering a range of values, enter single
+* or group values seperated by commas - ex. 1,10-20,30,33.
+*
+* PROGRAM PARAMETER SETTINGS START HERE ***************************************
+*------------------------------------------------------------------------------
+* INVALID HOME ACCOUNT TYPES
+*------------------------------------------------------------------------------
+*  Line must begin with "IHAT:"
+*
+*  Enter the account types which will exclude an account from being
+*  processed by this program.
+*
+*  If left blank, all open account types will be targeted
+*
+*  VALID VALUES: range of values from 0-9999
+*  DEFAULT: (left blank - account type not a factor)
+IHAT:
+*------------------------------------------------------------------------------
+* INVALID TARGET ACCOUNT, SHARE, LOAN AND EXTERNAL LOAN TYPES
+*------------------------------------------------------------------------------
+*  Line must begin with "ITAT:" (for accounts)
+*  Line must begin with "ITST:" (for shares)
+*  Line must begin with "ITLT:" (for loans)
+*  Line must begin with "ITET:" (for external loans)
+*
+*  Enter the Account, Share, Loan and or External Loan types which will exclude
+*  an Account, Share, Loan and/or External Loan from accessibility by this
+*  program. If left blank, Account, Share, Loan types and/or External Loan
+*  types will not be a consideration when analyzing target accounts.
+*
+*  VALID VALUES: range of values from 0-9999 (Account, Share, Loan)
+*                range of values from 1-8 (External Loan)
+*  DEFAULT: (left blank - Account, Share, Loan and/or External Loan
+*            types not a factor)
+ITAT:
+ITST:
+ITLT:
+ITET:
+*
+*------------------------------------------------------------------------------
+* EXCLUDE ACCOUNTS WITH INVALID SSNs
+*------------------------------------------------------------------------------
+*  Line must begin with "ISSN:"
+*
+*  Enter a list of invalid primary name record SSNs. List as a single
+*  comma-delimited list. Accounts where the primary member's SSN are
+*  found in this list will be skipped over. Enter up to 10 different
+*  SSNs. Use only numeric values.
+*
+*  VALID VALUES: Up to 10, comma delimited, 9-digit numeric values
+*  DEFAULT: [left blank - no invalid SSNs]
+ISSN:000000000,111111111,222222222,333333333,444444444,555555555,666666666,777777777,888888888,999999999
```

Comment:
Good list. Question. Isn't 123456789 Also a common SSN used for testing purposes, making 11 commonly used ones?

Reply:
Changed it to 12 (added 1234567890 & 987654321)